pkg/virt-controller/services/nodeselectorrenderer_test.go

```diff
@@ -0,0 +1,145 @@
+package services
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	v1 "kubevirt.io/api/core/v1"
+)
+
+var _ = Describe("Node Selector Renderer", func() {
+	var nsr *NodeSelectorRenderer
+
+	Context("scheduling pods ", func() {
+		When("no selectors are defined in the VMI spec", func() {
+			BeforeEach(func() {
+				nsr = NewNodeSelectorRenderer(emptySelectors(), emptySelectors())
+			})
+
+			It("the node requires the KubeVirt schedulable label", func() {
+				Expect(nsr.Render()).To(Equal(map[string]string{"kubevirt.io/schedulable": "true"}))
+			})
+
+			When("the dedicated CPU option is defined", func() {
+				BeforeEach(func() {
+					nsr = NewNodeSelectorRenderer(emptySelectors(), emptySelectors(), WithDedicatedCPU())
+				})
+
+				It("must be scheduled on nodes featuring the `cpumanager` label", func() {
+					Expect(nsr.Render()).To(HaveKeyWithValue("cpumanager", "true"))
+				})
+			})
+
+			When("the TSC timer option is defined", func() {
+				var aFewHertzios int64
+
+				BeforeEach(func() {
+					aFewHertzios = 123
+					nsr = NewNodeSelectorRenderer(emptySelectors(), emptySelectors(), WithTSCTimer(&aFewHertzios))
```

Comment:
nit: can use `pointer.Int(123)`

Reply:
yeah, I thought about that, but all in all, since this file doesn't use that dependency, it boils down to the exact same number of lines ... 

I think I'll keep it as is.